src/DataStructures/Tensor/Expressions/ScaleBy.hpp

```diff
@@ -0,0 +1,31 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function scale_by
+
+#pragma once
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+
+/*!
+ * \ingroup Functions
+ * \brief scales a tensor by a scalar
```

Comment:
This is ambiguous: the verb "to scale" applies to "scaling up" as well as "scaling down". (Oxford: "reduce or increase in size according to a common scale").

Suggested unambiguous names: `divide_by` or `divide_components_by`.